client/dashboard/app-ciab/index.tsx

```diff
@@ -9,16 +9,8 @@ boot( {
 	Logo,
 	supports: {
 		overview: false,
-		sites: {
-			deployments: false,
-			performance: false,
-			monitoring: false,
-			logs: false,
-			backups: false,
-			scan: false,
-			domains: false,
-			emails: false,
-		},
+		sites: false,
+		ciabSites: true,
```

Comment:
WDYT if we introduce a `sites.list` to control whether to show the Sites List? This way `app-ciab` would have `supports.sites.list: false` and `supports.sites.ciabList: true`.

My concern with `ciabSites` is that it's depending on the developer to know that ciabSites will reuse all of the site routes. It seems too much "inside baseball".

Reply:
Good point, but IMO `ciab-style` feels too similar to just having a `context: 'ciab'` 🫠